compose.yml

```diff
@@ -22,17 +22,22 @@ services:
       - ./backend/test:/src/app/test:ro
       - ./backend/shared:/src/app/shared:ro
     restart: unless-stopped
+    env_file:
+      - .env
     depends_on:
       - database
 
   database:
     container_name: database
+    command: postgres -c stats_temp_directory=/tmp
```

Comment:
I tried searching for what this does, couldn't anything within 5 minutes, could you give a short explanation on this?

Reply:
postgres stores so called temporary statistics data [^1][^2]
we are probably never going to use it, but with the default settings postgres did not have access permissions to the default location it tries to write these stats to. So this solves that (also kinda mentioned in a [commit message](https://github.com/Liryvian/ft_transcendence/pull/34/commits/dfaa2cefbb88e22ee76f7a683c52c5ce74ad84c4) but i 100% forgive you for not finding that..)

--
sources:
[^1]: https://postgresqlco.nf/doc/en/param/stats_temp_directory/
[^2]: https://www.postgresql.org/docs/14/runtime-config-statistics.html#GUC-STATS-TEMP-DIRECTORY

Reply:
Aaaaa okay yes thanks for elaborating!